_server_execute_js.js

```diff
@@ -7,26 +7,36 @@ module.exports = function() {
     */
     hybridVersion: function() {
       const ipcRenderer = require('electron').ipcRenderer;
+      const remote = require('electron').remote;
+      let pc = remote.getGlobal('pageCache');
```

Comment:
Addressed the comments in commits 6158051 and 69744b1. I also went back and checked/modified the other instances of `let`s and `==`s.